news/_posts/translation/2022-12-05-do-you-speak-prestashop-december-2022.md

```diff
@@ -0,0 +1,127 @@
+* [French](https://docs.prestashop-project.org/translating-prestashop/v/french-2/)
+
+If you wish to see new content added to this guide, let us know on Slack or on Crowdin and we’d be more than happy to help.
+Note that (as always) you can [contribute to translating the documentation](https://docs.prestashop-project.org/translating-prestashop/translating-prestashop-software-basics/translating-documentation) if you wish to add or edit some elements. 
+
+### Proofreaders still needed!
+
+We are still looking for proofreaders for the Spanish (Argentina), Galician and Flemish projects! So if you want to help us, reach me on either Crowdin or Slack, I’d be more than happy to guide you 🙌
+
+If you wish to become a proofreader for any other language, let me know! We are always actively looking for proofreaders 👏
+
+### 💕 Many thanks to newcomers
+
+Many people subscribe to the project(s) of their choice each month, but not so many start translating straight away. For that reason, we would like to thank [Stellageo](https://crowdin.com/profile/stellageo/activity) who rushed into the efforts immediately!
+
+## A few stats
+
+* **16 members joined the project this month.**
+* **17 members** were active on the project.
+* A total of **4,674 words have been translated** and **7,757 validated**.
+* All this in **17 different languages**.
+ 
+Thank you very much for your involvement! 
+
+## Top contributors
+ 
+A lot of you are working every day on Crowdin to have PrestaShop available in many languages, and PrestaShop cannot thank you enough for your dedication! Here are the most active translators and proofreaders in November 2022.
+ 
+#### Top 5 translators in November:
+ 
+| |Translator | Language | # Words
+|-|---------- | -------- | ----------------
+| 1. | adonis_cn | Chinese Simplified | 3,099
+| 2. | AmirRzd | Persian | 646
+| 3. | stellageo | Greek | 215
+| 4. | Monika (monikaraciunaite) | Lithuanian | 161
+| 5. | Rauno Riikman (weaver) | Finnish | 124
+
+#### Top 5 proofreaders in November:
+ 
+| | Proofreader | Language | # Words
+|-| ---------- | -------- | ----------------
+| 1. | adonis_cn | Chinese Simplified | 3,103
+| 2. | Rostyslav (config123) | Ukrainian | 2,451
+| 3. | AmirRzd | Persian | 646
+| 4. | Morten Borg (mortenborg) | Danish | 510
+| 5. | Cha (cafetango) | Chinese Traditional | 306
+
+
+Congratulations, and welcome to our new top contributors! :clap:
+ 
+Remember, you can see who has been contributing to our translation project thanks to the [Translators page](https://translators.prestashop.com/).
+ 
+## Complete translations
+ 
+#### Fully available languages
+ 
+At the end of November, PrestaShop 8 was fully available (= 100% translated and validated) in **10 languages**. That’s great! Thank you for your involvement. :tada:
+ 
+| Catalan | Chinese (CN) | Czech | Dutch |
+| French | Greek | Italian | Portuguese (BR) |
+| Spanish (ES) | Ukrainian | 
+
+Thank you all for your contribution! :muscle: 
+
+#### Almost there!
```

Comment:
Do we keep this part? We already have the "Proofreaders needed" section at the beginning (and it's not the same languages)